AnnoyancesFilter/sections/popups.txt

```diff
@@ -171,8 +171,8 @@ studentshare.org#$#body { overflow: visible!important; }
 internet-technologies.ru###popup-subscribe
 glutenfrei-bier.com###trustamiOverlay
 glutenfrei-bier.com##.alert
-essen-und-trinken.de##.pier12-popup 
-essen-und-trinken.de##.blocker
+geo.de,essen-und-trinken.de##.pier12-popup 
```

Comment:
I don't see it on the page
@BlazDT please take a look

Reply:
Same

Reply:
<details>
<summary>Here it is</summary>
<img src="https://user-images.githubusercontent.com/18536328/75991373-7163d800-5f07-11ea-8c15-fba29d2c09a6.png" />
</details>

Reply:
After a longer waiting time I see it now too.